_assets/scripts/extract_logs.go

```diff
@@ -12,22 +13,36 @@ import (
 
 var logPattern = regexp.MustCompile(`(?P<timestamp>[\d-]+T[\d:]+\.\d+Z).*(?P<action>sent-message:.*)\s+(?P<logData>{.*})`)
 
-func main() {
-	if len(os.Args) != 2 {
-		fmt.Println("Usage: go run script.go <filename>")
-		os.Exit(1)
-	}
+const (
+	filtersNotMatched = "filters did match"
+	filtersMatched    = "filters did not match"
+	storeNodeMessage  = "received waku2 store message"
+)
 
-	filename := os.Args[1]
-	file, err := os.Open(filename)
-	if err != nil {
-		fmt.Printf("Error opening file: %s\n", err)
-		os.Exit(1)
+func receivedMessageCountInfo(scanner *bufio.Scanner) {
+	fmt.Printf("Matching\tNot matching\tStore node\tLive\tTotal\n")
+	filtersNotMatchedCount := 0
+	filtersMatchedCount := 0
+	storeNodeReceivedMessageCount := 0
+
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		// Check if the line contains "sent-message"
```

Comment:
Small nit: we are checking only received messages here right, comment needs to be changed.